From: Kovid Goyal <kovid@kovidgoyal.net>
Date: Wed, 20 Oct 2021 12:03:59 +0530
Subject: Move a bunch of metadata into setup.cfg

---
 setup.cfg | 31 +++++++++++++++++++++++++++++++
 setup.py  | 28 ----------------------------
 2 files changed, 31 insertions(+), 28 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index fa1f353..2c219d9 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,3 +1,34 @@
+[metadata]
+name = html5-parser
+author = Kovid Goyal
+author_email = redacted@acme.com
+description = Fast C based HTML 5 parsing for python
+license =Apache 2.0
+url = https://html5-parser.readthedocs.io
+platforms = any
+classifiers = 
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Text Processing
+    Topic :: Text Processing :: Markup
+    Topic :: Text Processing :: Markup :: HTML
+    Topic :: Text Processing :: Markup :: XML
+
+[options]
+package_dir =
+   =src
+packages = html5_parser
+install_requires = 
+    chardet
+    lxml>=3.8.0
+
+[options.extras_require]
+soup = beautifulsoup4
+
 [flake8]
 max-line-length = 100
 
diff --git a/setup.py b/setup.py
index b17aa7f..1e9655a 100755
--- a/setup.py
+++ b/setup.py
@@ -42,36 +42,8 @@ class Test(Build):
             raise SystemExit(ret)
 
 
-CLASSIFIERS = """\
-Development Status :: 5 - Production/Stable
-Intended Audience :: Developers
-License :: OSI Approved :: Apache Software License
-Natural Language :: English
-Operating System :: OS Independent
-Programming Language :: Python
-Topic :: Text Processing
-Topic :: Text Processing :: Markup
-Topic :: Text Processing :: Markup :: HTML
-Topic :: Text Processing :: Markup :: XML
-"""
-
 setup(
-    name='html5-parser',
     version='{}.{}.{}'.format(*version),
-    author='Kovid Goyal',
-    author_email='redacted@acme.com',
-    description='Fast C based HTML 5 parsing for python',
-    license='Apache 2.0',
-    url='https://html5-parser.readthedocs.io',
-    download_url=(
-        "https://pypi.python.org/packages/source/m/html5-parser/"
-        "html5-parser-{}.{}.{}.tar.gz".format(*version)),
-    classifiers=[c for c in CLASSIFIERS.split("\n") if c],
-    platforms=['any'],
-    install_requires=['chardet', 'lxml>=3.8.0'],
-    extras_require={'soup': 'beautifulsoup4'},
-    packages=['html5_parser'],
-    package_dir={'': 'src'},
     cmdclass={'test': Test},
     ext_modules=[
         Extension(
